.schemas/config.schema.json

```diff
@@ -678,6 +678,35 @@
               }
             }
           ]
+        },
+        "retry": {
```

Comment:
We have another retry section here:

https://github.com/ory/oathkeeper/blob/2c310f95063233b2d48d105a476f674516336537/.schemas/config.schema.json#L823-L838

Let's make this a definition ([example](https://github.com/ory/oathkeeper/blob/2c310f95063233b2d48d105a476f674516336537/.schemas/config.schema.json#L7-L27)) instead and link it using `$ref` ([example](https://github.com/ory/oathkeeper/blob/2c310f95063233b2d48d105a476f674516336537/.schemas/config.schema.json#L203-L205))